Makefile

```diff
@@ -4,46 +4,26 @@
 
 IS_DOCKER_INSTALLED = $(shell which docker >> /dev/null 2>&1; echo $$?)
 
-# list only our namespaced directories
-PACKAGES = $(shell go list ./... | grep -v '/vendor/')
-
 # docker info
 DOCKER_REPO ?= litmuschaos
+DOCKER_REPO ?= litmuschaos
 DOCKER_IMAGE ?= chaos-exporter
 DOCKER_TAG ?= ci
 
-.PHONY: all
-all: format lint deps build test security-checks push 
-
-.PHONY: help
-help:
-	@echo ""
-	@echo "Usage:-"
-	@echo "\tmake all   -- [default] builds the chaos exporter container"
-	@echo ""
-
-.PHONY: format
-format:
-	@echo "------------------"
-	@echo "--> Running go fmt"
```

Comment:
I think go fmt is required, but based on the condition like is there is some change then it should fail.

Reply:
go fmt is in GitHub workflow, there is a comment point to the makefile, but I will remove.